modules/Bio/EnsEMBL/Variation/DBSQL/PublicationAdaptor.pm

```diff
@@ -296,21 +296,25 @@ sub store{
     $pub->{dbID} = $dbh->last_insert_id(undef, undef, 'publication', 'publication_id');
 
     ## link cited variants to publication
-    $self->update_variant_citation($pub) if defined $pub->{variants}->[0] ;
+    $self->update_variant_citation($source_attrib_id,$pub) if defined $pub->{variants}->[0] ;
 
 }
 
 sub update_variant_citation {
```

Comment:
This is a public method, suggest adding a new parameter after existing parameter not before.
Suggest adding documentation for this method